i2c/ina228_i2c/ina228_i2c.c

```diff
@@ -0,0 +1,125 @@
+#include <stdio.h>
+#include "pico/stdlib.h"
+#include "hardware/i2c.h"
+#include <math.h>
+
+// LED GPIOs
+#define EXT_LED_GPIO 2
+
+// INA228 integrated shunt resistance in Ohms
```

Comment:
Is this shunt resistor actually part of the INA228, or just part of the Adafruit breakout board? If the latter, then IMHO the word "integrated" here ought to be removed.